src/model/points-model.js

```diff
@@ -29,30 +29,80 @@ export default class PointsModel extends Observable {
     return this.#offers;
   }
 
-  updateEvent(updateType, update) {
+
+  async init() {
+    try {
+      this.#destinations = await this.#eventsApiService.destinations;
+
+      this.#offers = await this.#eventsApiService.offers;
+      const events = await this.#eventsApiService.events;
+      this.#events = events.map(this.#adaptToClient);
+
+    } catch(err) {
+      this.#events = [];
```

Comment:
Нужно сбрасывать все данные в пустые массивы, потому что в `try` ошибка может произойти на любом запросе и другие могут успеть инициализироваться

```suggestion
      this.#events = [];
      this.#destinations = [];
      this.#offers = [];
```